src/QuarkusContext.ts

```diff
@@ -84,6 +84,7 @@ export namespace QuarkusContext {
 
 interface Defaults {
   buildTool: string;
+  platformVersion: string;
```

Comment:
This can also be removed since we're not making use of the default value for now.